docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -5,3 +5,57 @@
 * `selenium.grid.username` - defines remote grid username
 * `selenium.grid.password` - defines remote grid password
 * `selenium.grid.host` - defines remote grid hostname
+
+== Known Issues
+
+Known issues mechanism allows separate fails and known issues existing within the project.
+
+[IMPORTANT]
+JSON file should contain a valid JSON. If JSON is not valid known issues will be ignored and warning about invalid file will be printed.
+
+=== Properties
+
+[cols="2,1,2", options="header"]
+|===
+|Property
+|Default value
+|Description
+
+|known-issue-provider.fileName
+|known-issues.json
+|Defines the location of the known issues file
+|===
+
+=== Known Issues file
+
+The file represents a collection of JSON elements describing known issues.
+
+.known-issues.json
+[source,json,subs=attributes+]
+----
+ {
+    "VVD-1": { <1>
+        "type": "Internal",<2>
+        "assertionPattern": "Then `1` is = `2`.+, <3>
+        "storyPattern": "Validate the Math",<4>
+        "scenarioPattern": "Validate equality",<5>
+        "stepPattern": "Then `1` is = `2`.+"<6>
+        "runtimeDataPatterns": { <7>
+            "mainPageUrl": ".*vividus.dev.*",
+        }
```

Comment:
it's better to make the example more explicit and use `variablePatterns`: https://github.com/vividus-framework/vividus/blob/master/vividus-tests/src/main/resources/known-issues.json#L12